src/ui/components/x-button/component-test.js

```diff
@@ -0,0 +1,24 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('x-button', 'Integration | Component | x button', {
+  integration: true
+});
+
+test('it renders', function(assert) {
+  // Set any properties with this.set('myProperty', 'value');
+  // Handle any actions with this.on('myAction', function(val) { ... });
+
+  this.render(hbs`{{x-button}}`);
```

Comment:
@GavinJoyce shouldn't this be {{mu-first-post:x-button}} ?

Reply:
or whatever the namespace syntax is 😖

Reply:
thanks, updated

Reply:
the test fails with `{{mu-first-post:x-button}}`:

<img width="1276" alt="screen shot 2018-01-25 at 19 45 52" src="https://user-images.githubusercontent.com/2526/35408855-7b4e31a2-0208-11e8-926c-d7b44f87e96c.png">

Reply:
So I think this might have something to do with the resolver 🤔 I'm not even sure if the work myself and @rwjblue did even touched on exporting non "main" components in the way that the resolver is expecting (I'm not even sure I'm saying it right there)

Reply:
I've just had a look back at what we're doing and I remember now 🎉 the dummy app in this addon is intended to be a test of the new MU addon *being included into a classic app*. The ember-cli version we're using makes use of the (broccoli-module-unification-reexporter)[https://github.com/ember-cli/ember-cli/blob/mu-addons/package.json#L54) plugin that re-exports the components as "non namespaced" components.

I don't know the best way to proceed 😕

Reply:
Thanks. My aim here was to discover what was needed for tests in an addon. I think I've learned enough to apply this to the `ember g component` blueprint